### DISCUSSION TITLE
Adwaita and Adwaita-Dark themes

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2339] [Show and tell]
This has been weighing on me for a bit...personally I think Adwaita is pretty ugly in that "try to please everyone and you don't please anyone," vaguely corporate-ish way, but it's functional and ubiquitous. I've completed LXQt themes, palettes, and even a couple of matching Openbox themes, which are available <a href=https://github.com/AzumaHazuki/lxqt-themes-adwaita>here</a>.

@tsujan, though I've been trying to keep different sets of themes to their own repositories, given that Adwaita is to GTK 3 and 4 what Clearlooks is to GTK2, would you and the other devs be interested in a PR for these into the main lxqt-themes repo? If nothing else, having these in the default set might ease the transition for those coming from Xfce, MATE, or GNOME...

Here are a couple of screenshots: 
![Adwaita-Dark-LXQt](https://user-images.githubusercontent.com/67122280/203228890-4f22c298-79ee-4f55-9adb-61782a4d7621.jpg)
![Adwaita-LXQt](https://user-images.githubusercontent.com/67122280/203228891-dc3aa3cb-e807-4508-9945-efb16176177d.jpg)

**Comment by tsujan** [comment]
I have no problem with any theme as far as it's self-consistent. People have different tastes, and it's good to have themes with different looks and feels.

Please,

 1. Be careful about foreground and background colors: they should both be defined for all widgets.
 2. Test your themes with light and dark palettes to check whether any color is missing. Restart the panel and runner after changing the palette to make sure that they're completely updated.

**Reply by AzumaHazuki** [reply]
Sorry for the late reply here! Work's been busy and today I was cooking up a storm for Thanksgiving.

So, are there missing/inconsistently-themed widgets in this set somewhere? I generally do the development in a Xephyr session inside my usual Plasma desktop, so it's easy to kill and restart without the overhead of a VM. Things *seem* to come up looking correct...? They're based on my Clearlooks .qss files, which are part of the main tree...
